config/routes.rb

```diff
@@ -989,6 +989,7 @@
         show_list
         tagging_edit
         tag_edit_form_field_changed
+        ems_container_form_fields
```

Comment:
With the restful routes changes you've made below in routes.rb, you do not need to specify routes for `new`, `edit` and `show` explicitly anymore, so you can get rid of them.
Refer https://github.com/ManageIQ/manageiq/pull/7808/commits/7925dc71b210be2b0b098828af6ae412c0fae12a for a list of all required changes for restful routes